src/Controls/samples/Controls.Sample/Pages/Controls/CollectionViewGalleries/CarouselViewGalleries/CarouselItemsGallery.cs

```diff
@@ -60,8 +60,10 @@ public class CarouselItemsGallery : ContentPage
 			carouselView.SetBinding(CarouselView.ItemsSourceProperty, nameof(_viewModel.Items));
 			carouselView.SetBinding(CarouselView.PositionProperty, nameof(_viewModel.CarouselPosition));
 
-			var absolute = new Microsoft.Maui.Controls.Compatibility.AbsoluteLayout();
-			absolute.Children.Add(carouselView, new Rectangle(0, 0, 1, 1), AbsoluteLayoutFlags.All);
+			var absolute = new Microsoft.Maui.Controls.AbsoluteLayout();
+			AbsoluteLayout.SetLayoutBounds(carouselView, new Rectangle(0, 0, 1, 1));
+			AbsoluteLayout.SetLayoutFlags(carouselView, AbsoluteLayoutFlags.All);
+			absolute.Add(carouselView);
```

Comment:
Not related directly with the content of the PR but, could we add more overrides to the Add method in Layouts to avoid set flag etc with different methods and lines?